org.eclipse.jdt.ui/ui/org/eclipse/jdt/internal/ui/javaeditor/codemining/CalleeJavaMethodParameterVisitor.java

```diff
@@ -165,7 +165,13 @@ private boolean skipParameterNameCodeMining(String[] parameterNames, List<?> arg
 	}
 
 	private boolean skipParameterNamesCodeMinings(IMethod method) {
+		IPreferenceStore store= JavaPlugin.getDefault().getPreferenceStore();
+		boolean showFromOneParameter = store.getBoolean(PreferenceConstants.EDITOR_JAVA_SHOW_ONE_PARAMETER);
```

Comment:
Do you really want to read from the store for each method?

Reply:
And in line 179 another preference store access?

Reply:
Would it be ok to leave that to another PR so that this one doesn't blow up?

Reply:
Yes, sure.